src/test/java/com/github/terma/jenkins/githubprcoveragestatus/CompareCoverageActionTest.java

```diff
@@ -82,20 +88,40 @@ public void postCoverageStatusToPullRequestAsComment() throws IOException, Inter
     }
 
     @Test
-    public void postResultAsStatusCheck() throws IOException, InterruptedException {
+    public void postResultAsSuccessfulStatusCheck() throws IOException, InterruptedException {
         prepareBuildSuccess();
         prepareEnvVars();
         prepareCommit();
+        prepareCoverageData(0.88f, 0.95f);
         coverageAction.setPublishResultAs("statusCheck");
-
```

Comment:
@lextiz just remove the unnecessary spaces in the empty line here and below. I am going to test the PR and merge if all good